readme.md

```diff
@@ -7,6 +7,31 @@ This repository contains the following:
 - **Microsoft Azure Provisioning device SDK for C#** to provision devices to Azure IoT Hub with .NET
 - **Microsoft Azure Provisioning service SDK for C#** to manage your Provisioning service instance from a back-end .NET application
 
+### Build status
+
+Due to security considerations, build logs are not publicly available.
```

Comment:
Should we showing this publicly? Is the Azure pipelines visible to everyone?

Reply:
The pipeline details/ logs are not visible publicly, you need an AzureDevOps account, and appropriate permissions. All this shows is the last build status for our prod and canary pipelines.

Reply:
I get that. Was just wondering if this is something of value in the public readme.

Reply:
It's a sort of motivating tool we use to strive for green builds. It's embarrassing when this badge is red all the time

Reply:
ha ha I was just thinking of the embarrassing part and hence the question :)

Reply:
Ah, got it! This is indicative of the stability of our code, it's pretty common for OSS projects to display this kind of info.